### PR TITLE
OD-137 [Fix] Apply text link appearance settings to anchors

### PR DESCRIPTION
**Pull request by ivandevupp**
@sofiiakvasnevska
## Issue
OD-137 https://weboo.atlassian.net/browse/OD-137
## Description
 Apply text link appearance settings to anchors
## Screenshots/screencasts
https://monosnap.com/file/MpBa8FwGlJxdZfkqeqmaMMnQlEVtyz
## Backward compatibility
This change is fully backward compatible.
## Reviewers 
@AndrRyaz @YaroslavOvdii

### CODE REVIEW

File: scss/core/base.scss [hunk 1]
```diff
@@ -10,37 +10,65 @@ TABLES
...
-  text-decoration: none;
+  font-weight: $linkFontWeight;
+  font-style: $linkFontStyle;
+  text-decoration: $linkFontDecoration;
 
   &:hover,
   &:focus,
   &:active:hover,
   &:active:focus {
+    font-family: $linkFontFamilyHover;
```
Comment (tonytlwu) [inline scss/core/base.scss:24]: For the hover and focus selectors, if we only keep `color` and `outline`, will that work? If not, what's overriding these properties?

Comment (ivandevupp) [reply scss/core/base.scss:24]: Yes, it will work. Should I leave in hover/focus only color with variable and `outline: none`?

Comment (tonytlwu) [reply scss/core/base.scss:24]: @ivandevupp Yes, please. This way the generated CSS file is smaller.

Comment (ivandevupp) [reply scss/core/base.scss:24]: Probably didn't get right your question, all options selected for text will work, but we need to customize text hover too 
image

Comment (tonytlwu) [reply scss/core/base.scss:24]: I see. Looks fine then.

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Medium
Affected: All text links
Verify: Text links are correctly styled according to appearance settings

Squash and merge: Use PR title

**Pull request merged**